src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Templates/Templates/VisualBasicWebAppTemplates/Startup.vb

```diff
@@ -0,0 +1,46 @@
+﻿' This Startup file Is based on ASP.NET Core New project templates And Is included
+' as a starting point for DI registration And HTTP request processing pipeline configuration.
+' This file will need updated according to the specific scenario of the application being upgraded.
+' For more information on ASP.NET Core startup files, see https://docs.microsoft.com/aspnet/core/fundamentals/startup
+
+Imports Microsoft.AspNetCore.Builder
+Imports Microsoft.AspNetCore.Hosting
+Imports Microsoft.Extensions.Configuration
+Imports Microsoft.Extensions.DependencyInjection
+Imports Microsoft.Extensions.Hosting
+
+Public Class Startup
+    Public Sub New(configuration As IConfiguration)
+        Me.Configuration = configuration
+    End Sub
+
+    Public ReadOnly Property Configuration As IConfiguration
+
+    ' This method gets called by the runtime. Use this method to add services to the container.
+    Public Sub ConfigureServices(services As IServiceCollection)
+        services.AddControllers()
+    End Sub
+
+    ' This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
+    Public Sub Configure(app As IApplicationBuilder, env As IWebHostEnvironment)
+
+        If (env.IsDevelopment()) Then
+            app.UseDeveloperExceptionPage()
+        Else
+            ' The default HSTS value Is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
+            app.UseHsts()
```

Comment:
Noticed that we don't use HSTS in the CSharp example. Do we have a preference?

Reply:
Not a strong one. I went that way because I was trying to keep the template code as small and basic as possible.

Reply:
removed to consolidate our approach as minimal